inspire/modules/workflows/workflows/process_record_arxiv.py

```diff
@@ -251,7 +251,7 @@ def get_record(cls, obj, **kwargs):
         This object will be used for indexing and be the basis for display
         in Holding Pen.
         """
-        if isinstance(obj.data, six.text_type):
+        if isinstance(obj.get_data(), six.text_type):
```

Comment:
This can stay as data (as invenio-workflows) has the fix to make sure `obj.data` is there